main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopTimeArrivalAndDepartureTimeValidator.java

```diff
@@ -93,4 +104,8 @@ public void validate(NoticeContainer noticeContainer) {
       }
     }
   }
+
+  private boolean isTimepoint(GtfsStopTime stopTime) {
+    return stopTime.timepoint() == GtfsStopTimeTimepoint.EXACT;
```

Comment:
Please use `.equals()` method for enums - this is a requirement of Google Java style.

> Both choices above produce the correct result. However, we prefer code that is not just correct, but self-evidently correct. It's best if the reader can know the code is safe from bugs without having to look up whether the type of e1 and e2 is an enum. This makes equals the better choice